Local Rancher server does not respect proxy variables HTTP_PROXY and HTTPS_PROXY

**What kind of request is this (question/bug/enhancement/feature request):** bug

**Steps to reproduce (least amount of steps as possible):**

1. Start rancher server _locally_ with the `HTTP_PROXY` and `HTTPS_PROXY` environment variables set

**Result**
Start up (1) reaches out to `https://releases.rancher.com/ui/latest2/index.html` and (2) ignores HTTP Proxy environment variables `HTTP_PROXY` and `HTTPS_PROXY` causing the request and UI to fail to load:

```
2019/07/16 14:41:40 [ERROR] failed to serve UI: Get https://releases.rancher.com/ui/latest2/index.html: dial tcp 104.24.17.51:443: connect: network is unreachable
```

--- a/server/ui/ui.go
+++ b/server/ui/ui.go
@@ -15,6 +15,7 @@ import (
 var (
 	insecureClient = &http.Client{
 		Transport: &http.Transport{
+			Proxy: http.ProxyFromEnvironment,
 			TLSClientConfig: &tls.Config{
 				InsecureSkipVerify: true,
 			},

